app/src/main/java/com/github/sdp_begreen/begreen/fragments/MapFragment.kt

```diff
@@ -241,7 +253,8 @@ class MapFragment : Fragment() {
         if (ContextCompat.checkSelfPermission(
                 requireContext(),
                 Manifest.permission.ACCESS_FINE_LOCATION
-            ) != PackageManager.PERMISSION_GRANTED) {
+            ) != PackageManager.PERMISSION_GRANTED
+        ) {
 
```

Comment:
The indentation here is really weird.

The only good indentation I could find is extracting the permission to a val so it display the code more naturally

eg:
`val permission = ContextCompat.checkSelfPermission(
            requireContext(),
            Manifest.permission.ACCESS_FINE_LOCATION
        )

        if (permission != PackageManager.PERMISSION_GRANTED) {

            requestPermissionLauncher.launch(Manifest.permission.ACCESS_FINE_LOCATION)

        } else {

            displayUserLocation()
        }`

Reply:
That's a good idea, I refactored it a tiny bit differently, tell me what you think

Reply:
I'm not convinced but idc